packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowFields.tsx

```diff
@@ -35,6 +38,9 @@ export class DetailsRowFields extends BaseComponent<IDetailsRowFieldsProps, IDet
                 cellStyleProps.cellRightPadding +
                 (column.isPadded ? cellStyleProps.cellExtraRightPadding : 0);
 
+          const { onRender = onRenderItemColumn } = column;
+          const cellContentsRender = onRender && !shimmer ? onRender(item, itemIndex, column) : getCellText(item, column);
```

Comment:
Can you error on verbosity and add parenthesis?

(onRender && !shimmer) ? ... : ...

or

onRender && (!shimmer ? ... : ...)

Reply:
I THIHK prettier strips things out ...

Reply:
> 
> 
> I THIHK prettier strips things out ...

prettier does remove the parens from the first case on commit

```tsx
const cellContentsRender = (onRender && !shimmer) ? onRender(item, itemIndex, column) : getCellText(item, column);
```

The second suggestion seems to clash with the intent of the ternary, but perhaps I'm misunderstanding.

```tsx
const cellContentsRender = onRender && (!shimmer ? onRender(item, itemIndex, column) : getCellText(item, column));
```

Reply:
Spoke with David offline, sticking with what we have. Resolving.